docs/docs/deploy/clever-cloud.md

```diff
@@ -0,0 +1,209 @@
+---
+sidebar_position: 4
+title: 💎 Clever Cloud
+---
+
+# Clever Cloud 💎
+
+[Clever Cloud](https://www.clever-cloud.com/) is a PaaS provider that enables you to start apps quickly using `git` while they manage the infrastructure, scalability, and run lifetime.
+
+## Prerequisites
+
+Before you get started, if you don't already have these, you'll need to create:
+
+- A [Clever Cloud Account](https://api.clever-cloud.com/v2/sessions/signup)
+- Add billing information
+
+:::caution
+As with any PaaS service you should check the pricing before continuing. Clever Cloud has a transparent pricing policy depending on the runtime type and size, see the [Clever Cloud Pricing Page](https://www.clever-cloud.com/pricing/).
+:::
+
+You might be able to do everything in the console, but for this tutorial, we will use the [Clever Cloud CLI](https://www.clever-cloud.com/doc/reference/clever-tools/getting_started/), so you'll have to install it on your computer.
+
+Then log in running:
+
+```bash
+clever  login
+```
+
+## Deploying
+
+### 1. Create the Clever Cloud application
```

Comment:
```suggestion
1. Create the Clever Cloud application
```

The other deployment documentations don't use the "###" header. Can you edit these headers to be consistent with the styling used in the other deployment documentations?

Reply:
Of course I can.
But I find it harder to follow, here are some screenshots with or without

For the index 

<img width="174" alt="Screenshot 2024-01-11 at 22 01 27" src="https://github.com/VeryGoodOpenSource/dart_frog/assets/2990298/367ea1e2-c38c-4357-bbd7-b70944d7594c">
<img width="236" alt="Screenshot 2024-01-11 at 22 01 56" src="https://github.com/VeryGoodOpenSource/dart_frog/assets/2990298/fb23a254-cfb9-4501-aad9-bbd484637d46">

And the content itself

<img width="382" alt="Screenshot 2024-01-11 at 22 01 35" src="https://github.com/VeryGoodOpenSource/dart_frog/assets/2990298/12d9d4b3-8621-4246-bf3d-9197a359d80c">
<img width="511" alt="Screenshot 2024-01-11 at 22 02 02" src="https://github.com/VeryGoodOpenSource/dart_frog/assets/2990298/640c119b-240f-4f80-a57c-384fa6370297">

As you wish, I understand the argument to make it look consistent, but maybe it can be done updating the other deployment documents ? (I can make another PR for this)

Reply:
I agree, the list view looks better. If you can update the other deployment documentation in a separate Pull Request that would be wonderful. What do you think about this @wolfenrain ?

Reply:
Of course ! 
It's up to you, if we keep on the `###` for the titles I'll change the `Congratulations` t `Enjoy your API in Clever Cloud` and make another PR for the other sections with pleasure to make it consistent

Reply:
I would love to have such an extra overview! That does make it more readable!

Reply:
Let's do this so. I'll update the title of this PR from `Congratulations` to `Enjoy your API in Clever Cloud` and make another one for consistency on the other pages.

Reply:
Up to date

Reply:
Thank you @quentin7b ! You 🪨 !